scripts/ci-notify-approval.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+LASTTAG=$(git describe --abbrev=0 --tags HEAD~1)
+GITLOG=$(git log --no-merges --format="%cd %s" --date=short $LASTTAG...HEAD | sed -e ':a' -e 'N' -e '$!ba' -e 's/\n/\\n/g')
```

Comment:
We're going to have to do some escaping here to avoid the $PAYLOAD JSON being invalid if a commit message contains a `"` char. I believe adding `sed 's/\"/\\"/g'` will achieve this i.e.:

```
GITLOG=$(git log --no-merges --format="%cd %s" --date=short $LASTTAG...HEAD | sed -e ':a' -e 'N' -e '$!ba' -e 's/\n/\\n/g' | sed 's/\"/\\"/g')
```